### PR TITLE
Update known 3rd parties

### PR DESCRIPTION
**Pull request by harahu**
Add listings for `plotly` and `pycryptodome`.

### CODE REVIEW

File: src/flake8_requirements/modules.py [hunk 1]
```diff
@@ -657,11 +657,13 @@
     "pillow": ["PIL"],
     "pillow-simd": ["PIL"],
     "pip-tools": ["piptools"],
+    "plotly": ["plotly", "_plotly_utils"],
```
Comment (arkq) [inline src/flake8_requirements/modules.py:660]: I can see that plotly exports more packages: https://github.com/plotly/plotly.py/blob/master/packages/python/plotly/setup.py#L521 All exported packages should be listed here (for brevity's sake `plotly.*` can be added as a single `plotly`).

**Review by arkq — CHANGES_REQUESTED** [review]
Also, please bump minor version number: https://github.com/arkq/flake8-requirements/blob/master/src/flake8_requirements/checker.py#L25. After merging this PR I will release new version.

File: src/flake8_requirements/modules.py (hunk 1 above)
Comment (harahu) [reply src/flake8_requirements/modules.py]: Good catch. Thanks.

**Comment by harahu** [comment]
> Also, please bump minor version number: https://github.com/arkq/flake8-requirements/blob/master/src/flake8_requirements/checker.py#L25. After merging this PR I will release new version.

I updated the minor version like you said. Sure you didn't mean the patch version?

**Comment by arkq** [comment]
(facepalm)... Yes, I was thinking about patch version component... Sorry for that. So, please update patch number only.

**Review by arkq — APPROVED** [review]

**Pull request merged**